x-pack/plugin/core/src/main/resources/monitoring-ent-search-mb.json

```diff
@@ -0,0 +1,717 @@
+{
+  "index_patterns": [".monitoring-ent-search-${xpack.stack.monitoring.template.version}-*"],
+  "version": ${xpack.stack.monitoring.template.release.version},
```

Comment:
Can you add a `_meta` map that includes `"managed": true` and a "description" field for what this template is for?

Reply:
Done! 👍

Reply:
It looks like you added one to the `monitoring-beats-mb.json` file, but didn't add one for this template yet?

Reply:
🤦  Sorry for that. I actually modified it now instead of another random file 🤦 

Thanks for catching this up, @dakrone 👍